ui/admin/app/components/form/host-catalog/static/index.hbs

```diff
@@ -72,25 +73,24 @@
     </form.fieldset>
 
     {{#if @model.isPlugin}}
+      <Rose::Separator />
       <form.fieldset>
         <:title>{{t 'resources.host-catalog.form.provider-details'}}</:title>
         <:body>
           <form.radioGroup
-            @name='aws-plugins'
+            @name='aws_plugin'
```

Comment:
Radio groups do not have names, do they?

Reply:
looks like they do have a [name](https://boundary-ui-storybook.vercel.app/?path=/docs/rose-form-radio-group--basic)

Reply:
Ah right it's the name for all of the radio fields in the group.  So should the name be `provider` or something?

Reply:
you are right. that makes sense here. updating now